README.md

```diff
@@ -37,5 +37,6 @@ $ python -m pytest
 Currently not fully type-checkable until we convert all our legacy code!
 
 ```shell
-mypy fal
+$ mypy fal run_fal.py
```

Comment:
Shouldn't we also run mypy on run_fal2.py while we are at it?

Reply:
well currently we can't even check `run_fal.py` because it's on legacy code. I'm actually gonna address this in the next PR so if you don't mind punting on this since it's just documentation it'd be appreciated.